src/routes/docs/advanced/platform/release-policy/+page.markdoc

```diff
@@ -0,0 +1,155 @@
+---
+layout: article
+title: Release policy
+description: Understand how Appwrite releases and versions its platforms and APIs.
+---
+We value the trust of developers in Appwrite as the backbone of their applications. 
+Our release policy is designed to provide developers with a reliable and consistent experience when using Appwrite. 
+We are committed to providing support for our API, SDKs, and product versions for a reasonable length of time, 
+and we follow industry-standard versioning protocols. 
+Appwrite will prioritize security updates and will release new versions immediately to fix any security vulnerabilities.
+
+# Schedule {% #schedule %}
+
+We work to release a new minor version of the product every quarter, which will include new features and enhancements. 
+
+We prioritize the timely release of patch versions with bug fixes and security updates on top of these feature releases, 
+and we make every effort to ensure that our releases are thoroughly tested and stable before they are made available to developers. 
+
+In rare cases where there are significant delays or changes to our release schedule, we will notify through our website, Discord server, 
+newsletter, and other communication channels.
+
+# Scope  {% #scope %}
+Appwrite will provide two phases of continued support for older versions of Appwrite.
+
+{% table %}
+* Phase {% width=100 %}
+* Scope
+---
+* Support
+* Receive continued bug fixes and security updates.
+---
+* Extended security support
+* Receives only security updates.
+{% /table %}
+
+# Support  {% #support %}
+Appwrite commits to the continued support of our software with extended support policies for security related fixes for older versions of Appwrite.
+Supported versions will continue to receive bug fixes and security updates. Extended security support versions will only receive security updates.
+{% table %}
+* Releases {% width=100 %}
+* Current Version 
+* Support 
+* Extended Security Support
+---
+* API
+* `v1`
+* 3 latest versions
+* 10 years
+---
+* SDK
+* [See list](https://appwrite.io/docs/sdks)
+* 5 latest versions
+* 10 years
+---
+* Self-hosted
+* `1.5.x`
+* Latest major version (`>= 1.x.x`)
+* 10 years
+---
+* Runtimes
+* [See list](https://appwrite.io/docs/products/functions/runtimes)
+* 24 months
+* Per vendor
+{% /table %}
+# API versions {% #api-versions %}
+
+We are committed to providing developers with a stable and reliable API. 
+Appwrite API versions don’t change very often but are reserved for breaking changes.
+
+Currently, the latest stable version of the Appwrite API is `/v1`. 
+We use this prefix in all our API endpoint paths to allow API versioning. 
+Any Appwrite version regardless of its version, is committed to **maintain backward compatibility** for each version of the API you decided to use as long as this API version is still under maintenance support.
```

Comment:
version, version, version, version 😅  Can we rewrite this line? Its not clear to me